### DISCUSSION TITLE
Daily 25/10

### DISCUSSION THREAD

**Discussion by NataliaAlvarezIspc** [grupo-ispc-2023-2-0/workflix-app#292] [Daily]
- ¿Qué hice ayer?
- ¿Qué hice o haré hoy?
- Impedimentos o dificultades

**Comment by Valentino399** [comment]
Ayer estuve haciendo el java para la conexión de la ruta del video para ponerlo en la app tuve muchos problemas y aun no me anda anoche me apareció un cartel diciendo que el video no está disponible hoy tratare de seguir con eso y ver si puedo solucionar este problema

**Comment by NataliaAlvarezIspc** [comment]
Ayer terminé con el dashboard de administradores, me demandó mucho tiempo ya que me tiraba error al tratar de loguearme como admin, además de la lentitud de la computadora. Finalmente era un error de procedimiento al crear el usuario admin, no aclarado. Estuve todo el día con este tema y no llegué a completar la daily de ayer. También estuve dando una mano a algunos compañeros con sus tareas, revisando la app y creando tareas de bugs que fui encontrando. Por la tarde participé en la clase sincrónica del instituto.
Hoy me asigné algunas tareas pendiente que voy a realizar. Por la tarde voy a participar del encuentro de proyecto y por la noche tengo clase de Inglés.

**Comment by BraianTroncoso** [comment]
Ayer estuve en clase y luego de clase ayude a varios de mis compañeros con problemas que le surgieron. Estuve realizando multiples tareas, solución de bugs, validaciones en Dashboards, traer recursos del back, implementación de APIS y funciones genéricas para que puedan ser re utilizadas. Estuve renegando casi 9hs porque no podia mostrar en el Perfil los atributos del back cuando vienen con un valor por default, ya que si ese usuario no tiene datos(que no tiene varios atributos porque cuando se registra no los completa) estos venian en NULL, por suerte despues de tanto esfuerzo logre validar en caso de que si o que no, y quedó perfecto sin bug alguno, seguí testeando la app y probando que todo funcione correctamente. Hoy tambien voy a seguir con tareas que nadie completó para seguir avanzando hasta terminar, voy a participar de la clase de hoy también.
Impedimentos los mismos de todas las dailys poca o nula participación, comunicación y la falta de tiempo para terminar con todo esto. Sumado a la Pc que no ayuda nada, cada 2 por 3 tengo que reiniciar y arrancar todos los software de 0.
